code/modules/mining/equipment_locker.dm

```diff
@@ -197,6 +197,8 @@
 		dat += text("Plastitanium Alloy (Titanium + Plasma): <A href='?src=[UID()];alloytype1=/obj/item/stack/sheet/mineral/titanium;alloytype2=/obj/item/stack/sheet/mineral/plasma;alloytypeout=/obj/item/stack/sheet/mineral/plastitanium'>Smelt</A><BR>")
 	if(glassstack && plasmastack && min(glassstack.amount, plasmastack.amount))
 		dat += text("Plasma Glass (Glass + Plasma): <A href='?src=[UID()];alloytype1=/obj/item/stack/sheet/glass;alloytype2=/obj/item/stack/sheet/mineral/plasma;alloytypeout=/obj/item/stack/sheet/plasmaglass'>Smelt</A><BR>")
+	if(glassstack && titaniumstack && min(glassstack.amount, titaniumstack.amount))
+		dat += text("Titanium Glass (Titanium + Glass): <A href='?src=[UID()];alloytype1=/obj/item/stack/sheet/glass;alloytype2=/obj/item/stack/sheet/mineral/titanium;alloytypeout=/obj/item/stack/sheet/titaniumglass'>Smelt</A><BR>")
```

Comment:
Going to hold off on merging this until the ORM refactor goes through.

On the plus side, by waiting, to implement new alloys, you only have to add them to here: **https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/research/designs/smelting_designs.dm**

Reply:
@Fox-McCloud Thanks for doing the refactor, makes this a whole lot simpler